tests/test_runners/test_googletest.py

```diff
@@ -43,5 +43,5 @@ def test_record_empty_dir(self):
         path = 'latest/gtest_*_results.xml'
         result = self.cli('record', 'tests', '--session', self.session,
                           'googletest', path)
-        self.assertEqual(result.output.rstrip('\n'), "No matches found: {}".format(path))
+        self.assertIn("No matches found: {}".format(path), result.output.rstrip('\n'))
```

Comment:
Q: why do we need to change this line?

Reply:
Thank you for asking this question. It's because I called client.run() in `record_tests method`, not `load_report_files`. I've fixed it.

[074d915](https://github.com/launchableinc/cli/pull/846/commits/074d915c1cb79cdbfacad92c7e307e4f46025e94)